spec/form/work_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Sufia::Forms::WorkForm do
```

Comment:
This should be testing `Sufia::Forms::GenericWorkForm` instead.  `WorkForm` is an abstract class.